### PR TITLE
SPEC-1395 FLE test fixes

### PR DESCRIPTION
**Pull request by divjotarora**

### CODE REVIEW

File: source/client-side-encryption/tests/README.rst [hunk 1]
```diff
@@ -487,5 +495,5 @@ The corpus test exhaustively enumerates all ways to encrypt all BSON value types
    - If ``allowed`` is true, decrypt the value with ``client_encryption``. Decrypt the value of the corresponding field of ``corpus_encrypted`` and validate that they are both equal.
    - If ``allowed`` is false, validate the value exactly equals the value of the corresponding field of ``corpus`` (neither was encrypted).
 
-9. Repeat steps 1-8 with a local JSON schema. I.e. amend step 4 to configure the schema on ``client_encrypted`` and ``client_encryption`` with the ``schema_map`` option.
+9. Repeat steps 1-8 with a local JSON schema. I.e. amend step 4 to configure the schema on ``client_encrypted`` with the ``schema_map`` option.
```
Comment (kevinAlbs) [inline source/client-side-encryption/tests/README.rst]: Good catch.

File: source/client-side-encryption/client-side-encryption.rst [hunk 2]
```diff
@@ -676,7 +676,7 @@ An encrypted MongoClient MUST attempt to auto decrypt the results of all
 commands.
 
 Drivers MUST raise an error when attempting to auto encrypt a command if
-the maxWireVersion is less than 8.
+the maxWireVersion is less than 8. The error message MUST contain "Auto-encryption requires a minimum MongoDB version of 4.2".
```
[the remark this answers is not in the document]
Comment (jyemin) [reply source/client-side-encryption/client-side-encryption.rst]: +1

File: source/client-side-encryption/tests/getMore.json [hunk 3]
```diff
@@ -142,7 +142,7 @@
             "batchSize": 2,
             "filter": {}
           },
-          "results": [
+          "result": [
```
[the remark this answers is not in the document]
Comment (divjotarora) [reply source/client-side-encryption/tests/getMore.json]: Done in next commit.

File: source/client-side-encryption/client-side-encryption.rst (hunk 2 above)
[the remark this answers is not in the document]
Comment (divjotarora) [reply source/client-side-encryption/client-side-encryption.rst]: Filed https://jira.mongodb.org/browse/SPEC-1463 and I'll make a PR for that once this is done.

File: source/client-side-encryption/tests/README.rst (hunk 1 above)
Comment (divjotarora) [reply source/client-side-encryption/tests/README.rst]: Thanks.

File: source/client-side-encryption/client-side-encryption.rst (hunk 2 above)
[the remark this answers is not in the document]
Comment (divjotarora) [reply source/client-side-encryption/client-side-encryption.rst]: Also created a PR for this: https://github.com/mongodb/specifications/pull/653

**Pull request merged**